### PR TITLE
Fix up imports for Python3 compatibility on the tests

### PR DESCRIPTION
**Pull request by hornc**
Latest tests fail on this code:
```
infogami/infobase/core.py:61: in <module>
    class Text(unicode):
E   NameError: name 'unicode' is not defined
```
https://travis-ci.org/hornc/infogami/jobs/577183152

Which is the subject of the comment in #39 to be done in another PR.

This was done as a quick experiment to see what was required to at least get the tests loading in Python 3. #39 should perhaps be merged first, I don't think there are conflicts, but #39 was first, and this will be easier to redo.

### CODE REVIEW

File: infogami/infobase/common.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import config
 import web
 
-from core import *
-from utils import *
+from infogami.infobase import config
+from infogami.infobase.core import *
+from infogami.infobase.utils import *
```
Comment (cclauss) [inline infogami/infobase/common.py]: Can we lose the wildcard (*) imports?  PEP8 discourages their use and they disable all _undefined name_ checking.

**Comment by cclauss** [comment]
Yes. Let's land #39 first and then will generate a __unicode__ PR.

**Comment by hornc** [comment]
@cclauss I have merge #39 and rebased this against master.
Tests fail on ```infogami/infobase/core.py:61: in <module>
    class Text(unicode):
E   NameError: name 'unicode' is not defined```

I'll look at removing those wildcard imports.

**Comment by cclauss** [comment]
Unicode fixes are in #41

**Comment by cclauss** [comment]
Yes!  Delete these lines https://github.com/internetarchive/infogami/blob/master/infogami/infobase/utils.py#L8-L20 and delete this import.  all() and any() are builtin to all supported versions of Python.

**Comment by cclauss** [comment]
Time to step it up folks...  Today, there are __100 days__ until Python 2 end of life and the Python Software Foundation and Python Core Team have been _very clear_ that they [will not even fix known security vulnerabilities](https://www.python.org/doc/sunset-python-2) discovered after 1/1/2020..

**Comment by cclauss** [comment]
infogami/infobase/writequery.py needs __from infogami.infobase import account, common__ like in #42

**Pull request merged**